runtime/src/main/kotlin/konan/Annotations.kt

```diff
@@ -61,6 +61,13 @@ public annotation class Used
 @Retention(AnnotationRetention.SOURCE)
 public annotation class Throws(vararg val exceptionClasses: KClass<out Throwable>)
 
+/**
+ * Class is immutable and is frozen by default.
+ */
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.BINARY)
+annotation class Immutable
```

Comment:
This annotation is unchecked and thus unsafe. Shouldn't it be internal yet?

Reply:
OK